src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -189,6 +189,6 @@
         default-view (get-in tabs [0 :view-id])
         view-id      (reagent/atom default-view)]
     [react/view {:style styles/flex}
-     [status-bar/status-bar {:type :modal}]
+     [status-bar/status-bar {:type :modal-white}]
```

Comment:
is this change related to the fact that the android status toolbar is black in the transactions screen ?

Reply:
yes, there is a bug on android, status bar is gray on modal screens, i decided to make it white on ios (as designed) and black on android (because bug ), black looks better than gray